File: week10/CodeKata10.py
class Engineer:
    def __init__(self, androidExpert, iosExpert, frontendExpert, backendExpert):
        self.androidExpert = androidExpert
        self.iosExpert = iosExpert
        self.frontendExpert = frontendExpert
        self.backendExpert = backendExpert

PERCENTAGE_ASSIGNED_ANDROID = 0.2;
PERCENTAGE_ASSIGNED_IOS = 0.25;
PERCENTAGE_ASSIGNED_FRONTEND = 0.2;
PERCENTAGE_ASSIGNED_BACKEND = 0.2;

def distributeEngineersInProjects(engineers):
    #TO-DO

File: week10/CodeKata10.swift
class Engineer {
    var androidExpert: Bool
    var iosExpert: Bool
    var frontendExpert: Bool
    var backendExpert: Bool
    
    init(androidExpert:Bool, iosExpert:Bool, frontendExpert:Bool, backendExpert:Bool) {
        self.androidExpert = androidExpert;
        self.iosExpert = iosExpert;
        self.frontendExpert = frontendExpert;
        self.backendExpert = backendExpert;
    }
}

let percentageAsignedAndroid = 0.2;
let percentageAsignedIos = 0.25;
let percentageAsignedFrontend = 0.2;
let percentageAsignedBackend = 0.2;

func distributeEngineersInProjects(engineers: [Engineer]) -> [Engineer] {
    // TO-DO
}